README.md

```diff
@@ -506,6 +514,17 @@ This allows to identify build errors caused by smalltalkCI updates and
 therefore helps to improve the service. It is possible to opt-out by using the
 `--no-tracking` option.
 
+### Pharo-specific Options
+
+Pharo added some specific options to customise its builds: 
+
+```yml
+env:
+  global:
+  - PHARO_VM=stable|latest              # choose stable/latest VM (default: stable)
```

Comment:
Hi @estebanlm,
We've tried hard so far to stay away from adding dialect-specific options via `env`. The main reason is that we'd need to support them forever as soon as users start using them. That's why I am hesitating here.

Also, I don't understand how your proposed approach solves the problem with regard to itimer/threaded only being available on Linux. Setting `env: global: LINUX_HEARTBEAT=threaded` would also affect Mac builds, but it would be ignored.

I'm really just looking for the best (and generic) way to add support for manual vm selection and I still imagine something like this at the moment:
``` yml
jobs:
  include:
    - smalltalk: Pharo-7.0
      smalltalk_vm: Pharo-7.0+latest
    - smalltalk: Pharo64-7.0
      smalltalk_vm: Pharo64-7.0+stable+itimer
    - smalltalk: Squeak-5.1
    - smalltalk: Squeak-5.1
      smalltalk_vm: OpenSmalltalkVM+latest
```
Everything with `+` are vm options, so `Pharo-7.0+stable+itimer` would be the same as `Pharo-7.0+itimer+stable`.